### DISCUSSION TITLE
Umbraco Forms: Add warning/notification when user has unsaved changes in a Form and/or Workflow

### DISCUSSION THREAD

**Discussion by mennomout** [umbraco/Umbraco-CMS#13835] [Features and ideas]
**Summary:** Add a warning/notification when an user has unsaved changes in Umbraco Forms and tries to navigate to another page in the CMS.

**Current behavior:** While having unsaved changes to a form or workflow I can navigate to another form without warning an lose all my changes.

**Desired behavior:** While having unsaved changes to a form or workflow I first get a notification/warning telling the user they will lose their unsaved changes.

**Suggested implementation:** Similar to having unsaved changes in a doctype and/or content.

**Comment by nul800sebastiaan** [comment]
Hey @mennomout, thanks for the suggestion but this won't be seen by the Forms team. make sure to add it to the Forms tracker here:  https://github.com/umbraco/Umbraco.Forms.Issues/issues/new/choose

**Comment by lassefredslund** [comment]
I'll close this idea, as it now exists in the Forms issue tracker here: https://github.com/umbraco/Umbraco.Forms.Issues/issues/979
